Dockerfile

```diff
@@ -33,12 +33,18 @@ ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 # Note that the ordering of these operations is intentional to minimize package footprint
 WORKDIR /home/nginx/.nvm
 ENV NVM_DIR=/home/nginx/.nvm
-COPY deploy/docker/production/install_nginx.sh /
+COPY deploy/docker/production/install_nginx.sh /install_nginx.sh
+
+# Temporarily replacing
+#     curl -sSL https://install.python-poetry.org | POETRY_HOME=/opt/venv python - && \
+# with
+#     pip install poetry==1.1.15
+# because of a problem with wheel compatibility in 1.2.0. Need to debug that later. -kmp 31-Aug-2022
 RUN apt-get update && apt-get upgrade -y && \
     apt-get install -y --no-install-recommends vim emacs net-tools ca-certificates build-essential \
     gcc zlib1g-dev postgresql-client libpq-dev git make curl libmagic-dev && \
     pip install --upgrade pip && \
-    curl -sSL https://install.python-poetry.org | POETRY_HOME=/opt/venv python - && \
+    pip install poetry==1.1.15 && \
```

Comment:
I would consider pinning this via env variable similar to others above, like nvm/node version

Reply:
I'll do that in a later PR. I'll queue a ticket for that. For now I just want it to match what we're doing in cgap-portal, and I didn't do it that way there.